_data/apps.yml

```diff
@@ -1035,3 +1035,12 @@
     - "research"
     - "spreadsheet"
   icon: "dataproofer.png"
+-
+  name: "Electron TimeTable"
+  description: "TimeTable in your menubar!"
+  website: "https://github.com/willyb321/electron-menubar-timetable"
+  repository: "https://github.com/willyb321/electron-menubar-timetable"
+  keywords:
+    - "Timetable"
+    - "menubar"
+  icon: "icon.png"
```

Comment:
This needs to be the name of the icon you are adding in this pull request, in this case `TTTIcon.png`